### PR TITLE
Update Zyxel PMG3000-D20B

### PR DESCRIPTION
**Pull request by Andrea055**
- Add some commands

I'm not sure that fw_setenv command error is a bug.

### CODE REVIEW

File: _ont/ont-zyxel-pmg3000-d20b.md [hunk 1]
```diff
@@ -104,6 +104,19 @@ SW_VER1:0xabcedf
 ```
 {% include alert.html content="it's better to enter the software version in hex format, all lowercase precedeed by 0x." alert="Note"  icon="svg-info" color="blue" %}
 
+## See link speed
+This SFP have HSGMII enabled by default, link_status=5 HSGMII 2.5Gbit, link_status=5 SGMII 1Gbit:
```
Comment (simonebortolin) [inline _ont/ont-zyxel-pmg3000-d20b.md:108]: are you sure about this in lantiq stick should be `link_status=4` SGMII 1Gbit.

in any case you should put `link_status` inside inline code with ``

File: _ont/ont-zyxel-pmg3000-d20b.md [hunk 2]
```diff
@@ -104,6 +104,19 @@ SW_VER1:0xabcedf
...
+```
+linuxshell
+onu lanpsg 0
+```
+
+## Connection state
+To see the connection state use this command, curr_state=5 O5 state, curr_state=1 all states other than O5:
```
Comment (simonebortolin) [inline _ont/ont-zyxel-pmg3000-d20b.md:115]: you should put `curr_state=` inside inline code with ``

File: _ont/ont-zyxel-pmg3000-d20b.md [hunk 3]
```diff
@@ -159,6 +172,7 @@ To restore the default combination you can try following [this method](https://g
 - [Not working with Broadcom BCM57810S](https://github.com/xvzf/zyxel-gpon-sfp/issues/10)
 - Issue on IPv6 discovery. But we are not sure if is a edge case of a particular ISP or not
 - [Some stick have a custom password](https://github.com/xvzf/zyxel-gpon-sfp/issues/6)
+- fw_setenv command doesn't work
```
Comment (simonebortolin) [inline _ont/ont-zyxel-pmg3000-d20b.md:175]: does not work or is not present, it seemed to me the second one

**Review by simonebortolin — CHANGES_REQUESTED** [review]

File: _ont/ont-zyxel-pmg3000-d20b.md (hunk 3 above)
Comment (Andrea055) [reply _ont/ont-zyxel-pmg3000-d20b.md:175]: admin@SFP:~# fw_setenv sgmii_mode 5
Cannot parse config file: No such file or directory
Error: environment not initialized

File: _ont/ont-zyxel-pmg3000-d20b.md (hunk 1 above)
Comment (Andrea055) [reply _ont/ont-zyxel-pmg3000-d20b.md:108]: sorry, I was wrong to write

File: _ont/ont-zyxel-pmg3000-d20b.md (hunk 3 above)
Comment (simonebortolin) [reply _ont/ont-zyxel-pmg3000-d20b.md:175]: OK is not present

Comment (simonebortolin) [reply _ont/ont-zyxel-pmg3000-d20b.md:175]: Like the new firmware of fs.com stick

File: _ont/ont-zyxel-pmg3000-d20b.md [hunk 4]
```diff
@@ -105,14 +105,14 @@ SW_VER1:0xabcedf
...
-This SFP have HSGMII enabled by default, link_status=5 HSGMII 2.5Gbit, link_status=5 SGMII 1Gbit:
+This SFP have HSGMII enabled by default, link_status=`5` HSGMII 2.5Gbit, link_status=`4` SGMII 1Gbit:
 ```
 linuxshell
 onu lanpsg 0
 ```
 
 ## Connection state
-To see the connection state use this command, curr_state=5 O5 state, curr_state=1 all states other than O5:
+To see the connection state use this command, curr_state=`5` O5 state, curr_state=`1` all states other than O5:
```
Comment (simonebortolin) [inline _ont/ont-zyxel-pmg3000-d20b.md:115]: Mmmh no. 
```
`curr_state=5`
```

**Review by simonebortolin — CHANGES_REQUESTED** [review]

**Review by simonebortolin — APPROVED** [review]

**Comment by simonebortolin** [comment]
since fw_printev is not a bug but an absence, you can still write it otherwise in a new PR. thank you very much for contributing

**Pull request merged**